aten/src/ATen/Declarations.cwrap

```diff
@@ -1935,8 +1935,8 @@
       output: True
     - THTensor* self
     - THTensor* other
-    - arg: long dim
-      default: -1
+    - arg: c10::optional<int64_t> dim
```

Comment:
I think it's better to just port this function to ATen.  The issue with what you've done here is that you've added an implicit mapping from "int?" to "c10::optional<int64_t>" -- that mapping is enforced if you put it in native_functions, but not here.  Instead of building yet another mapping, it's just cleaner to put it in ATen.

Reply:
so you can just delete this entry.

Reply:
So because I moved just CPU implementation I left this entry, but changed it to int64_t and removed default value.